src/parser_urdf_TEST.cc

```diff
@@ -22,6 +22,20 @@
 #include "sdf/sdf.hh"
 #include "parser_urdf.hh"
 
+/////////////////////////////////////////////////
+/// Check that _a contains _b
+static bool contains(const std::string &_a, const std::string &_b)
+{
+  return _a.find(_b) != std::string::npos;
+}
+
+/////////////////////////////////////////////////
+/// Check that _a does not contain _b
+static bool not_contains(const std::string &_a, const std::string &_b)
```

Comment:
nit: Use camel case. Also, can you put this in `test_utils.hh` as well?

Reply:
https://github.com/gazebosim/sdformat/pull/1238/commits/cb6d540769af94cd41bcc6bc17de74d96deea39d